hooks/useOrders.ts

```diff
@@ -38,6 +38,21 @@ export const useBoughtProducts = (
   const supabase = useSupabaseClient();
   const { selectedOrder } = useOrderHistory();
 
+  const getProducts = async (boughtProducts: BoughtProduct[]) => {
```

Comment:
If I put this outside of the hook, and use it as another hook I get some errors when calling it inside here.